src/main/resources/setting.fxml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import com.jfoenix.controls.JFXDrawer?>
+<?import com.jfoenix.controls.JFXToggleButton?>
+<?import javafx.scene.control.Button?>
+<?import javafx.scene.control.Label?>
+<?import javafx.scene.control.Separator?>
+<?import javafx.scene.control.TextField?>
+<?import javafx.scene.layout.AnchorPane?>
+<?import javafx.scene.layout.HBox?>
+<?import javafx.scene.text.Font?>
+
+
+<AnchorPane maxHeight="-Infinity" maxWidth="-Infinity" minHeight="-Infinity" minWidth="-Infinity" prefHeight="800.0" prefWidth="1000.0" xmlns="http://javafx.com/javafx/17" xmlns:fx="http://javafx.com/fxml/1" fx:controller="gui.view.SettingsController">
+   <children>
+      <AnchorPane fx:id="mainBackground" layoutX="42.0" prefHeight="800.0" prefWidth="950.0" style="-fx-background-color: #E5E5E5;" AnchorPane.leftAnchor="50.0">
+         <children>
+            <HBox layoutX="130.0" layoutY="148.0" prefHeight="76.0" prefWidth="319.0" AnchorPane.leftAnchor="130.0" AnchorPane.topAnchor="148.0">
```

Comment:
Though I thought having HBox is redundant, we can still keep it.
Also I notice that your main page and setting page can fit the screen when I maximize the window (though it move to the left top corner, it's still better than other pages we have).
So if you have time, it would be good to fix that bugs for other pages